aplib.net-demo/Assets/Scripts/InputManager.cs

```diff
@@ -59,14 +61,21 @@ private void Start()
         _playerActions.Jump.canceled += _ => _playerMovement.OnJumpUp();
         _playerActions.Respawn.performed += _ => _playerRespawn.Respawn();
         _playerActions.UseItem.performed += _ => _inventory.ActivateItem();
-        _playerActions.NextItem.performed += _ => equipmentInventory.MoveNext();
+        _playerActions.NextItem.performed += _ =>
+        {
+            equipmentInventory.MoveNext();
+
+            _playerAnimator.SetTrigger("SwitchWeapon");
+        };
         _playerActions.PreviousItem.performed += _ => equipmentInventory.MovePrevious();
         _playerActions.SwitchItem.performed += _ => _inventory.SwitchItem();
         _playerActions.Fire.performed += _ =>
         {
             if (equipmentInventory.HasItems)
             {
                 equipmentInventory.CurrentEquipment.UseEquipment();
+
+                _playerAnimator.SetTrigger("PlayerAttack");
```

Comment:
Should probably not be done here, the shooting animation still plays if the player has no ammo

Reply:
This has been changed